Project.toml

```diff
@@ -17,6 +17,7 @@ Mustache = "ffc61752-8dc7-55ee-8c37-f3e9cdd09e70"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 PrettyTables = "08abe8d2-0d0c-5749-adfa-8a2ac140af0d"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
```

Comment:
Can you check if the TOML entry requires a compat entry?

Reply:
Turns out that TOML is part of the standard library starting in Julia 1.6. I'll open an issue to remove this once we drop support for 1.5. I added a compat entry here but am not sure if it is required. Let me know if this was wrong.